packages/v4/patternfly-docs/components-data.json

```diff
@@ -0,0 +1,354 @@
+  },
+  "input-group": {
+    "illustration": "./images/component-illustrations/input-group.png",
+    "summary": "An <b>input group</b> includes multiple input types, like a text input and a button, in one control."
+  },
+  "jump-links": {
+    "illustration": "./images/component-illustrations/jump-links.png",
+    "summary": "<b>Jump links</b> allow users to navigate to sections within a page."
+  },
+  "label-group": {
+    "illustration": "./images/component-illustrations/label-group.png",
+    "summary": "Use a <b>label group</b> when you have multiple <Link to='https://www.patternfly.org/v4/components/label'>labels</Link> to display at once. Label groups can be oriented either horizontally or vertically and can optionally be named and dismissable."
+  },
+  "label": {
+    "illustration": "./images/component-illustrations/label.png",
+    "summary": "The <b>label</b> component allows users to add specific element captions for user clairty and convenience."
+  },
+  "list": {
+    "illustration": "./images/component-illustrations/list.png",
+    "summary": "A <b>list</b> component embeds a formatted list (bulleted or numbered list) into page content."
+  },
+  "login-page": {
+    "illustration": "./images/component-illustrations/login-page.png",
+    "summary": "A <b>login page</b> allows a user to gain access to an application by entering their username and password or by authenticating using a social media login."
+  },
+  "masthead": {
+    "illustration": "./images/component-illustrations/masthead.png",
+    "summary": "A <b>masthead</b> contains global properties such as logotype, navigation and settings in an organized fashion and it is accessible from all parts of the application. All pages should share a common masthead."
+  },
+  "menu-toggle": {
+    "illustration": "./images/component-illustrations/menu-toggle.png",
+    "summary": "The <b>menu toggle</b> component pairs with the <Link to='https://www.patternfly.org/v4/components/menu/design-guidelines'>menu</Link> component to create composable dropdown and select menus. Using a menu toggle with a menu enables you to create custom component configurations not supported by the standard dropdown or select components. However, from a design perspective, the dropdown, select, and menu toggle components are all visually the same. For more information about this component's usage, see the <Link to='https://www.patternfly.org/v4/components/dropdown/design-guidelines'>dropdown</Link>, <Link to='https://www.patternfly.org/v4/components/select/design-guidelines'>select</Link>, and <Link to='https://www.patternfly.org/v4/components/menu/design-guidelines'>menu</Link> component guidelines."
```

Comment:
Could we note that the menu toggle component can `pair with the Menu component OR the Panel component to create more customizable dropdown and select implementations`?

I also wonder if the sentence about the design perspective is appropriate in this section?